ServiceNow Service Catalog v2.5.0 updates, Washington DC updates. (#567)
hashicorp/terraform-docs-common: website/docs/cloud-docs/integrations/service-now/service-catalog-terraform/index.mdx

necessarily the same person, this documentation refers to a **ServiceNow Admin**
and a **Terraform Admin**.

> First, the Terraform Admin will configure your HCP Terraform organization with a

Reviewer: ```suggestion
First, the Terraform Admin configures your HCP Terraform organization with a
```

> dedicated team for the ServiceNow integration, and obtain a team API token for

Reviewer: ```suggestion
dedicated team for the ServiceNow integration, and obtains a team API token for
```

> that team. The Terraform Admin will provide your organization name, the team API

Reviewer: Can we make this a list? It's a pretty dense list as is and splitting it out would make it more readable:
```suggestion
that team. The Terraform Admin provides the following to your ServiceNow admin:
* An Organization name
* A team API token
* The hostname of your HCP Terraform instance
* Any available no-code modules or version control repositories containing Terraform configurations 
* Other required variables
```

## Configure HCP Terraform

Before installing the ServiceNow integration, the Terraform Admin will need to
perform the following steps to configure and gather information from HCP
Terraform.

1. Either [create an
   organization](/terraform/cloud-docs/users-teams-organizations/organizations#creating-organizations)
   or choose an existing organization where ServiceNow will create new
   workspaces.
   **Save the organization name for later.**
2. [Create a team](/terraform/cloud-docs/users-teams-organizations/teams) for that
   organization called "ServiceNow", and ensure that it has [permission to
   manage
   workspaces](/terraform/cloud-docs/users-teams-organizations/permissions#manage-all-workspaces).
   You do not need to add any users to this team.
   [permissions-citation]: #intentionally-unused---keep-for-maintainers
3. On the "ServiceNow" team's settings page, generate a [team API
   token](/terraform/cloud-docs/users-teams-organizations/api-tokens#team-api-tokens).
   **Save the team API token for later.**
4. If you are using the [VCS flow](/terraform/cloud-docs/integrations/service-now/service-catalog-terraform/service-catalog-config#vcs-configuration): 

>     1. If you haven't yet done so, [connect a VCS

Reviewer: I think you can omit the second sentence here because you link out to a place that contains more information. I tried shortening this a bit for brevity but please feel free to push back if anything doesn't feel good:
```suggestion
    1. Ensure your Terraform organization is [connected a VCS provider](/terraform/cloud-docs/vcs). Repositories that are connectable to HCP Terraform workspaces can also be used as workspace templates in the ServiceNow integration.
```

Reviewer: (Leaving this as one comment rather than spacing it out over the lines because it's easier to read, but if you accept, this is a reminder to delete the lines below!)

    provider](/terraform/cloud-docs/vcs) to this Terraform organization. This provider will include one or more repositories with Terraform configurations. Any repository that could be connected to manually-created HCP Terraform workspace can also be used as a workspace template in the ServiceNow integration.

>     2. On your organization's VCS provider settings page (Settings > VCS Providers), find the "OAuth Token ID" for the VCS provider or providers you will be using with the ServiceNow integration. HCP Terraform uses the OAuth token ID to identify and authorize the VCS provider. **Save the OAuth token ID for later.**

Reviewer: Adding bold for what I assume are UI elements:
```suggestion
    2. On your organization's VCS provider settings page (**Settings** > **VCS Providers**), find the OAuth Token ID for the VCS provider(s) that you intend to use with the ServiceNow integration. HCP Terraform uses the OAuth token ID to identify and authorize the VCS provider. **Save the OAuth token ID for later.**
```

    3. Identify the VCS repositories in the VCS provider containing Terraform configurations that the ServiceNow Terraform integration will deploy. Take note of any Terraform or environment variables used by the repositories you select. Save the Terraform and environment variables for later.
5. If using the [no-code flow](/terraform/cloud-docs/integrations/service-now/service-catalog-terraform/service-catalog-config#no-code-module-configuration), create one or more no-code modules in the private registry of your HCP Terraform. **Save the no-code module names for later.**
6. Provide the following information to the ServiceNow Admin:

>     1. The organization name 

Reviewer: Can all of the following items please be in an unordered list for visual distinction?
```suggestion
    * The organization name 
```

     the hostname is `https://app.terraform.io`. Be sure to include "https://"
     before the hostname.
   - Set API Team Token to the Terraform Cloud team API token.

>    - (Optional) To use the [MID Server](https://docs.servicenow.com/bundle/vancouver-servicenow-platform/page/product/mid-server/concept/mid-server-landing.html), 

Author: @sheneska please update the URL to not contain "vancouver" in it as it will break sometime in the future when Vancouver is deprecated. Whenever we share SNOW documentation links, let's use the "share" button > "Copy latest version URL" to copy a link that is not tied to any specific version.

>      select the checkbox and type the `name` in the `MID Server Name` in the field.

Author: @sheneska should it be just "in the `MID Server Name` field"?
